src/frontend/citizen-portal/src/app/components/dispute-submit-success/dispute-submit-success.component.ts

```diff
@@ -7,7 +7,7 @@ import { TicketTypePipe } from '@shared/pipes/ticket-type.pipe';
 import { AppRoutes } from 'app/app.routes';
 import { DisputeResourceService } from 'app/services/dispute-resource.service';
 import { DisputeService } from 'app/services/dispute.service';
-import { truncate } from 'fs';
```

Comment:
remove code instead of commenting out  unless you are certain it will be needed in near future